configs/1-3B.yml

```diff
@@ -3,7 +3,7 @@
    # parallelism settings ( you will want to change these based on your cluster setup, ideally scheduling pipeline stages
    # across the node boundaries )
    "pipe_parallel_size": 1,
-   "model_parallel_size": 1,
+   "model_parallel_size": 2,
```

Comment:
Let's not change the existing configs

Reply:
The llama config changes are fine, to be clear

Reply:
accidentally left it there when testing.